doc/development/pull-api.md

```diff
@@ -0,0 +1,79 @@
+# Pull-API: Terminology
+
+
+## Selector
+
+Quoted vector holding one or several of the following expressions:
+1) wildcard
+   - pulls all datoms for entity id
+   - e.g. * -> {:db/id 1 :name "Petr" :child [{:db/id 2}]}
```

Comment:
with some backticks you can get `monospace` for code snippets here.

Reply:
Thanks for the tip, I will adjust the formatting